src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -79,6 +79,7 @@ void Graph::CreateGraph(NET &net, const ExtensionManager::Ptr& extMgr,
 
     rtParamsCache = std::make_shared<MultiCache>(config.rtCacheCapacity);
     sharedMutex = mutex;
+    rtScratchPad = std::make_shared<DnnlScratchPad>(getEngine());
```

Comment:
With this change we add new strong dependency of CPU graph on oneDNN backend. 
This is not merge stopper here, but smt we need to resolve in future. At first glance there should be generic Scratchpad (or lets say Memory pool) and DNNL one is just a specific impl. 

@maxnick FYI

Reply:
I would say this `DnnlScratchPad`  class do not use any dnnl_specific stuff, except `dnnl;:engine` which is still a part of CPU plugin memory class. `DnnlMemoryDesc` is OV CPU plugin internal class. But, you are right, `getScratchPadMem` could have `MemoryDescPtr` argument instead `DnnlMemoryDescPtr` and it should be changed. In all other matters this do not introduce additional one dnn dependencies instead of its name. In reality, the entire scratchpad is just a ov::intel_cpu::Mmemory object, which is stored on the graph level.